Assets/LeapMotion/Scripts/LeapServiceProvider.cs

```diff
@@ -305,5 +351,62 @@ public class LeapServiceProvider : LeapProvider {
 
       dest.CopyFrom(source).Transform(leapTransform);
     }
+
+    //Precull-Latching Functions
+    protected virtual void OnEnable() {
+      Camera.onPreCull -= LateUpdateHandTransforms;
+      Camera.onPreCull += LateUpdateHandTransforms;
+      _transformArray[0] = Matrix4x4.identity;
+      _transformArray[1] = Matrix4x4.identity;
+      Shader.SetGlobalMatrixArray("_handTransforms", _transformArray);
```

Comment:
The name of the matrix array should be a public constant defined at the top instead of hardcoded.  Especially since Unity does not throw any errors when you set properties that don't exist.